avocado/plugins/runner_nrunner.py

```diff
@@ -227,7 +227,8 @@ async def _update_status(self, job):
             elif status == 'finished':
```

Comment:
Just a typo fix on commit message: "insturemted"

Reply:
Oh, thanks for catching that. I will fix it right away :+1: